src/patternfly/layouts/Page/styles.scss

```diff
@@ -0,0 +1,73 @@
+// URL.com/guidelines#layout
+@import "../../patternfly-utilities";
+
+$pf-l-page__sidebar-Width:    15rem !default;
+$pf-l-page--breakpoint:       $pf-global--breakpoint--sm !default;
```

Comment:
This variable `$pf-l-page--breakpoint:` is not following the system. Since in this case there is no property we can call it `MediaMinWidth` and add a comment explaining why